recipe/patch_yaml/snappy.yaml

```diff
@@ -6,3 +6,13 @@ then:
   - replace_depends:
       old: snappy >=1.1.7,<1.1.8.0a0
       new: snappy >=1.1.7,<2.0.0.0a0
```

Comment:
@jjerphan Can you also adjust this line, too?

Reply:
Sure. How would you like to adjust them exactly?